Python/Python.sublime-syntax

```diff
@@ -2159,7 +2159,7 @@ contexts:
 
   builtin-types:
     - match: '{{builtin_types}}'
-      scope: support.type.python
+      scope: storage.type.python
```

Comment:
This will definitely take some getting used to when seeing these used as constructors … I understand the argument, but I fundamentally disagree with scoping identifiers the same for the type definition/declaration and the namespace/constructor use cases. I've mentioned that sufficiently in #1842, but we haven't followed up with something that we could generally agree on.

That said, I wonder if it wouldn't be better to rather stay on the defensive side here and only change the scopes of long-untouched parts of syntaxes once we have a decision to not disrupt user workflows potentially twice.